test/e2e/reliability/reliability_test.go

```diff
@@ -115,54 +115,51 @@ var _ = Describe("test reliability", Label("reliability"), Serial, func() {
-			GinkgoWriter.Printf("check pod ip in the ippool：%v\n", allipRecord)
-
-			// get nodename list
-			nodeMap := make(map[string]bool)
-			for _, item := range podlist.Items {
-				GinkgoWriter.Printf("item.Status.NodeName:%v\n", item.Spec.NodeName)
-				nodeMap[item.Spec.NodeName] = true
-			}
-			GinkgoWriter.Printf("get nodeMap is：%v\n", nodeMap)
-
-			// send cmd to reboot node and check node until ready
-			ctx, cancel := context.WithTimeout(context.Background(), time.Minute*2)
-			defer cancel()
-			readyok, err := common.RestartNodeUntilReady(frame, nodeMap, time.Minute, ctx)
-			Expect(readyok).Should(BeTrue(), "timeout to wait node ready\n")
-			Expect(err).ShouldNot(HaveOccurred())
+			var daemonSetName string = "ds" + tools.RandomName()
 
-			// check pod ready and ip assign ok
-			podlistready, errip2 := frame.WaitDeploymentReadyAndCheckIP(podName, namespace, time.Second*30)
-			Expect(errip2).ShouldNot(HaveOccurred())
+			// Create Deployment
+			GinkgoWriter.Printf("Create Deployment %v/%v \n", namespace, podName)
+			dep := common.GenerateExampleDeploymentYaml(podName, namespace, replicas)
+			Expect(frame.CreateDeployment(dep)).NotTo(HaveOccurred(), "Failed to create Deployment")
 
-			// after reboot node check ip exists in ipppool
-			allipRecord2, _, _, errpool := common.CheckPodIpRecordInIppool(frame, defaultv4, defaultv6, podlistready)
-			Expect(allipRecord2).Should(BeTrue())
-			Expect(errpool).ShouldNot(HaveOccurred())
-			GinkgoWriter.Printf("check pod ip in the ippool：%v\n", allipRecord2)
+			// Wait for the deployment to be ready and check that the IP assignment is correct
+			podList, err1 := frame.WaitDeploymentReadyAndCheckIP(podName, namespace, time.Second*30)
+			Expect(err1).ShouldNot(HaveOccurred())
 
-			// delete Deployment
-			errdel := frame.DeleteDeployment(podName, namespace)
-			Expect(errdel).NotTo(HaveOccurred(), "failed to delete Deployment %v/%v \n", podName, namespace)
+			// Check if the IP exists in the IPPool before restarting the node
+			ClusterDefaultIPv4IPPool, ClusterDefaultIPv6IPPool, err2 := common.GetClusterDefaultIppool(frame)
+			Expect(err2).ShouldNot(HaveOccurred())
+			isRecord1, _, _, err3 := common.CheckPodIpRecordInIppool(frame, ClusterDefaultIPv4IPPool, ClusterDefaultIPv6IPPool, podList)
+			Expect(isRecord1).Should(BeTrue())
+			Expect(err3).ShouldNot(HaveOccurred())
+			GinkgoWriter.Printf("Pod IP recorded in IPPool %v,%v \n", ClusterDefaultIPv4IPPool, ClusterDefaultIPv6IPPool)
 
+			// Send a cmd to restart the node and check the cluster until it is ready
+			ctx, cancel := context.WithTimeout(context.Background(), time.Minute*2)
+			defer cancel()
+			err4 := common.RestartNodeUntilClusterReady(ctx, frame, podList.Items[0].Spec.NodeName)
+			Expect(err4).NotTo(HaveOccurred(), "Execution of cmd fails or node/Pod is not ready, error is: %v \n", err4)
+
+			// Create daemons to circumvent the situation where some pods are still not running after a node reboot
```

Comment:
//after all node reboot , create daemonset for checking spiderpool service ready, in case this rebooting test case influence on later test case